test/utils.js

```diff
@@ -142,7 +142,7 @@ export const graphqlQuery = async (query, variables, remoteUser, schema = schema
  * @param {object} variables - Variables to use in the queries and mutations. Example: { id: 1 }
  * @param {object} remoteUser - The user to add to the context. It is not required.
  */
-export async function graphqlQueryV2(query, variables, remoteUser) {
+export async function graphqlQueryV2(query, variables, remoteUser = null) {
```

Comment:
Why add a default `null` for this argument?

Reply:
In some places we call this function with only two arguments `graphqlQueryV2(query, variables)`. It's a way making the query without logging in so it's valid, but Typescript would complain that this function expects a third arguments. This `= null` is there to tell Typescript that this third argument is optional.